docs/getting_started.md

```diff
@@ -2,10 +2,10 @@
 
 Tye is a tool that makes developing, testing, and deploying microservices and distributed applications easier. Project Tye includes a local orchestrator to make developing microservices easier and the ability to deploy microservices to Kubernetes with minimal configuration.
 
-## Installing tye
+## Installing Tye
 
 1. Install [.NET Core 3.1](<http://dot.net>).
-2. Install tye via the following command:
+1. Install tye via the following command:
```

Comment:
I think it is still the second step, so "2." makes sense!

Reply:
git diff is going crazy showing this diff.
I checked the rich diff and it looks fine!

Reply:
![image](https://user-images.githubusercontent.com/1625492/120870924-2b088900-c54f-11eb-9535-c56e18f4fe3a.png)

Reply:
Markdown lets you use `1.` for every step and dynamically generates the correct number when shown, which makes it less likely that you'll end up with a mis-numbered list after future edits.  I try to switch to that format whenever I happen to see it.

Reply:
That's really smart! I didn't know that!